ArtOfIllusion/src/artofillusion/texture/ProceduralTexture2D.java

```diff
@@ -385,22 +294,22 @@ public void disposePreview(Object preview)
     UIUtilities.findWindow((MaterialPreviewer) preview).dispose();
   }
 
-  /** Determine whether the procedure may contain Parameter modules. */
+  /** Determine whether the procedure may contain View Angle modules. */
 
   @Override
-  public boolean allowParameters()
+  public boolean allowViewAngle()
```

Comment:
You've also switched the positions of `allowViewAngle()` and `allowParameters()`. This makes the diff harder to read and understand. Again, is there a reason for this?